RockLib.Configuration/RockLibConfigurationBuilderExtensions.cs

```diff
@@ -46,8 +47,30 @@ public static IConfigurationBuilder AddRockLib(this IConfigurationBuilder builde
             builder = builder.AddJsonFile(jsonConfigPath, optional: true);
 
             if (!string.IsNullOrEmpty(supplementalJsonConfigPath))
+            {
                 builder = builder.AddJsonFile(supplementalJsonConfigPath, optional: true);
+            }
+           
+            return builder;
+        }
+
+        /// <summary>
+        /// Adds support for .Net Framework applications to pull in App or Web.config AppSettings values.
+        /// </summary>
+        /// <param name="builder">The <see cref="IConfigurationBuilder"/> to add to.</param>
+        /// <returns>The <see cref="IConfigurationBuilder"/></returns>
+        public static IConfigurationBuilder AddAppOrWebConfig(this IConfigurationBuilder builder)
+        {
+
+#if NET451
```

Comment:
Need to move the `#if` outside of this method so that it doesn't exist in .NET Standard.